### PR TITLE
Draft Feature/model device

### PR DESCRIPTION
**Pull request by fabiankay**

### CODE REVIEW

**Comment by fabiankay** [comment]
ensure merge with main
update helium_api_wrapper endpoints

@g4challenge @elauser: maybe we can harmonize pipeline requirements with "helium_api_wrapper"
f.e. docstrings in files cause mypy errors

**Comment by g4challenge** [comment]
The mypy would now be comparably to the api-wrapper. I quickly fixed pre-commits and some of flake8 config, as well as tried to fix safety, mypy and tests.

The tests are not successful at the moment, please find a suitable way of testing before merging.

**Comment by elauser** [comment]
@fabiankay as an action item for todays planning we could add the steps needed to manually execute the pre-commit hooks and mypy to the readme.md I can take over that task if needed.

**Comment by fabiankay** [comment]
updates in helium_api_wrapper needed to work. Waiting for release.

**Comment by simonpfeifhofer** [comment]
@fabiankay would it be possible to extend the README and add how to select the model? (env-var, parameter, ...)

File: .env.sample
Comment (simonpfeifhofer) [inline .env.sample:8]: Would be nice to add to the README which models exist and how to select one.

File: src/helium_positioning_api/Models.py
Comment (simonpfeifhofer) [inline src/helium_positioning_api/Models.py:144]: At least trilateration would be nice to have it as part of this PR.

**Review by simonpfeifhofer — CHANGES_REQUESTED** [review]

**Comment by fabiankay** [comment]
> @fabiankay would it be possible to extend the README and add how to select the model? (env-var, parameter, ...)

Will be added to final PR

File: src/helium_positioning_api/Models.py
Comment (fabiankay) [reply src/helium_positioning_api/Models.py:144]: We planned adding trilateration in a new branch but will do it here.
Setting PR to draft for now

**Comment by knoll-fabio** [comment]
I believe there is no way to publish the sphinx documentation build via Github workflows on Read The Docs. To automatically deploy the documentation, we need to create a webhook in the Github repository, which is provided in the `Builds` section of the corresponding project on Read the Docs. Unfortunately, I don't have permission to access the webhook settings. @simonpfeifhofer could you take care of this please?

**Pull request merged**
